skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
```

Comment:
This is probably caused by this one here: https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/__init__.py#L13 <https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/__init__.py#L13>

Move that import into .selem and import label from there to `__init__.py`. Makes sense?